src/view/application/index.jsx

```diff
@@ -1,9 +1,9 @@
 import Navigator from '../navigator';
 
 const Application = () => (
-  <>
+  <div className='h-screen min-h-screen w-screen'>
```

Comment:
Revert this file and apply the h-screen class directly for the body html tag into your style.css file